Author: Adam Majer <amajer@suse.de>
Date: Wed Jan 25 12:13:10 CET 2017
Summary: Don't use no longer existant -mt boost libraries

All libraries are multi-threaded and have been for a long time.

Index: REL_3_1_4/src/Makefile.am
===================================================================
--- REL_3_1_4.orig/src/Makefile.am
+++ REL_3_1_4/src/Makefile.am
@@ -3,4 +3,4 @@
 INCLUDES            =  -I$(top_builddir) -I$(top_srcdir)
 bin_PROGRAMS = sfftobmp
 sfftobmp_SOURCES = my_getopt.c cmdline.cpp codes.cpp common.cpp decoder.cpp input.cpp main.cpp output.cpp
-sfftobmp_LDADD = @TIFF_LIB@ @JPEG_LIB@ -lboost_filesystem-mt -lboost_system
+sfftobmp_LDADD = @TIFF_LIB@ @JPEG_LIB@ -lboost_filesystem -lboost_system
